addon-test-support/index.js

```diff
@@ -0,0 +1,28 @@
+import QUnit from 'qunit';
+import { hasRunAllDisposables } from 'ember-lifeline';
+
+const FAILED_ASSERTION_MESSAGE =
+  'One or more objects registered disposables that were not correctly disposed of. Please ensure that objects correctly run their registered disposables by calling `runDisposables` in the `destroy` method of the object.';
+let setupTestDone = false;
+
+export default function setupEnsureLifelineDisposablesRun(hooks) {
```

Comment:
`setupLifelineValidation`? 🚲 🏠

Reply:
What does validation mean to the end user?